network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -271,6 +271,7 @@ class Petition(TranslatableMixin, CTA):
     ]
 
     class Meta(TranslatableMixin.Meta):
+        ordering = ['-id']
```

Comment:
Since snippets do not seem to have a "created" or "created_date" column, I am ordering them by their unique ID's which are assigned upon creation. That way we can order them by newest to oldest

Reply:
that's basically the only open for snippets, so: approved =P